README.md

```diff
@@ -394,7 +394,7 @@ This is what you should expect from Netdata:
 
 ### Integrations
 
--   **time-series dbs** - can archive its metrics to **Graphite**, **OpenTSDB**, **Prometheus**, **AWS Kinesis**, **MongoDB**, **JSON document DBs**, in the same or lower resolution (lower: to prevent it from congesting these servers due to the amount of data collected). Netdata also supports **Prometheus remote write API** which allows storing metrics to **Elasticsearch**, **Gnocchi**, **InfluxDB**, **Kafka**, **PostgreSQL/TimescaleDB**, **Splunk**, **VictoriaMetrics** and a lot of other [storage providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+-   **time-series dbs** - can archive its metrics to **Graphite**, **OpenTSDB**, **Prometheus**, **AWS Kinesis**, **MongoDB**, **JSON document DBs**, in the same or lower resolution (lower: to prevent it from congesting these servers due to the amount of data collected). Netdata also supports **Prometheus remote write API** which allows storing metrics to **Elasticsearch**, **Gnocchi**, **InfluxDB**, **Kafka**, **PostgreSQL**, **TimescaleDB**, **Splunk**, **VictoriaMetrics**, and a lot of other [storage providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
```

Comment:
Hi @joelhans , 
I verified that you changed `**PostgreSQL/TimescaleDB**` to `**PostgreSQL**, **TimescaleDB**`, this is not officially real, we do not have PostgreSQL exporter yet, the original text is due a reference present in TimescaleDB:

```
Powered by PostgreSQL

Leverage the reliability, maturity, and operational efficiency of one of the world's most popular databases.
```
For more details, please access  https://www.timescale.com/

Reply:
@vlvkobal I was reading TimescaleDB and I verified that is works with SQL, did we already try to use this specific exporter connecting direct to PostgreSQL?

Reply:
Interesting. Well, I can restore that to the way it was. Or, maybe write **PostgreSQL via TimescaleDB**?

Reply:
As I understand, TimescaleDB can use the same protocol as PostgreSQL. We don't provide a direct connector. Only through Prometheus Remote Write API.

Reply:
Hi @thiagoftsm — I reverted this to its original form, so it reads **PostgreSQL/TimescaleDB** again.